jwk/jwk_test.go

```diff
@@ -2070,3 +2070,25 @@ func TestGH730(t *testing.T) {
 	require.NoError(t, set.AddKey(key), `first AddKey should succeed`)
 	require.Error(t, set.AddKey(key), `second AddKey should fail`)
 }
+
+func TestECDSAPEM(t *testing.T) {
```

Comment:
**suggestion (non-blocking):** Include a similar reciprocal PEM encode/decode test for each type of JWK you support.